### DISCUSSION TITLE
en/implementing-google-adsense-performance/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#120] [General]
# en/implementing-google-adsense-performance/

Improve your Hugo website's performance with lazy loaded Google Adsense ads. This method minimizes impact on user experience by delaying ad loading with a script.

https://dariusz.wieckiewicz.org/en/implementing-google-adsense-performance/

**Comment by Salomens** [comment]
I am troubled by GA4, thank you very much for your article.
